### DISCUSSION TITLE
关于Cef控件不支持播放Html中视频的问题

### DISCUSSION THREAD

**Discussion by xiaobin-shi** [netease-im/NIM_Duilib_Framework#497] [Q&A]
cef加载了一个网页，这个网页使用<audio>控件播放视频，但是cef不支持。如何升级依赖的cef，请问是否有示例或者教程能提供吗

**Comment by nmgwddj** [comment]
CEF 官方有非常详细的构建流程和脚本，我们在这个仓库中讨论可能不是特别合适。CEF 的默认构建脚本中对 ffmpeg 的配置是不编译 GPL 协议代码的，要开启这部分代码需要你手工修改编译代码中的头文件开启一些编解码的支持，同时因 GPL 协议的限制，您必须开源引用了该代码的内容。具体请参考 CEF 官方。
